arcgis-ios-sdk-samples/Content Display Logic/Views/AppInfoView.swift

```diff
@@ -0,0 +1,168 @@
+                        Section(footer: Text(footer)) {
+                            LinkRow(link: link)
+                        }
+                    }
+                } else {
+                    Section {
+                        LinkRow(link: link)
+                    }
+                }
+            }
+            // Powered-by links section.
+            Section(header: Text("Powered By"), footer: Text(copyrightFooter)) {
+                ForEach(poweredBys) { link in
+                    LinkRow(link: link)
+                }
+            }
+        }
+    }
+}
+
+private extension AppInfoView {
+    struct LinkRow: View {
+        let link: Link
+        
+        var body: some View {
+            Button(action: {
+                UIApplication.shared.open(link.url)
+            }, label: {
+                Text(link.title)
+            })
+        }
+    }
+    
+    struct AppInfo: Identifiable {
+        let title: String
+        let detail: String
+        
+        var id: String { title }
+    }
+    
+    struct Link: Equatable, Identifiable {
+        let title: String
+        let url: URL
+        let subtitle: String?
+        
+        var id: String { title }
+        
+        init(title: String, url: URL, subtitle: String? = nil) {
+            self.title = title
+            self.url = url
+            self.subtitle = subtitle
+        }
+    }
+    
+    enum Strings {
+        /// The manually assigned marketing version, e.g. "100.13.0.1".
+        static let appMarketingVersionNumber = "100.13.0.1"
+        /// Name of this app bundle, i.e. "ArcGIS Runtime SDK Samples".
+        static let appName = Bundle.main.object(forInfoDictionaryKey: "CFBundleName") as? String ?? ""
+        /// An end-user string representation of the ArcGIS Runtime SDK for iOS
+        /// version shipped with the app, e.g. "100.13.0 (3355)".
+        static let ArcGISSDKVersionString = String(format: "%@ (%@)", sdkVersionNumber, sdkBuildNumber)
+        /// App version string representation for internal usage.
+        static let appInternalVersionString = String(format: "%@ (%@)", appVersionNumber, appbuildNumber)
+        /// App version number, e.g. "100.13.0".
+        private static let appVersionNumber = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String ?? ""
+        /// App build number, e.g. "20211214".
+        private static let appbuildNumber = Bundle.main.object(forInfoDictionaryKey: kCFBundleVersionKey as String) as? String ?? ""
+        /// SDK version number.
+        private static let sdkVersionNumber = Bundle.agsBundle?.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String ?? ""
+        /// SDK build number.
+        private static let sdkBuildNumber = Bundle.agsBundle?.object(forInfoDictionaryKey: "CFBundleVersion") as? String ?? ""
+    }
+    
+    enum URLs {
+        static let developers = URL(string: "https://developers.arcgis.com/ios/")!
```

Comment:
Mike mentioned in the review that "developer site" is more of an internal term. Should we change this name to be `sdkHomePage`?

Reply:
I'll leave it open.